### DISCUSSION TITLE
[Feature request] Custom Tag Color (added more requests)

### DISCUSSION THREAD

**Discussion by Manniala** [zanllp/infinite-image-browsing#443] [Ideas]
Would you be so kind to let us custom color out tags, could for instance be done next to where you do the Hotkey, either by just placing   #ff00ff for instance, or a real color picker.

[EDIT] Optional give people a option to have the color, represented in the pull-down menu also (pressing Hearth symbol), might be a fainted version of the color, not sure..

**Comment by Manniala** [comment]
[ADD]
And a custom sort order, either by dragging them in Search (where you also adds them), or settings next to Hotkey, where you can put in a Humber representing the sort order.

**Comment by Manniala** [comment]
[Added request]
Search for Images without tag, i would like if there was a way to search for images that does NOT have a tag yet, if it is possible, (i have not figured out how yet.)

**Reply by zanllp** [reply]
You can use Fuzzy Search.

**Reply by zanllp** [reply]
<img width="479" alt="image" src="https://github.com/zanllp/sd-webui-infinite-image-browsing/assets/25872019/3fb0a1a4-34da-4e38-aed1-1fd6f85c3af3">

**Reply by Manniala** [reply]
Never figured out how to use Fuzzy Search. :)

**Reply by zanllp** [reply]
Just like this, input a part of the EXIF or filename.


https://github.com/zanllp/sd-webui-infinite-image-browsing/assets/25872019/f51235f0-cab7-46c7-823c-66e3e3b0fdfa
